### PR TITLE
File platform: implemented experiment execution (batch and status, etc.)

### PR DESCRIPTION
**Pull request by ZDu-IDM**
This PR implemented:
- generated batch files: batch.sh, run_simulation.sh and _run.sh
- support singularity
- support two ways to run simulations:
   - sequence
   - parallel
- simulation status (similar to Slurm Platform)

**Extras work**
- implement CLI utility to check running status
- to be done in next PR

### CODE REVIEW

File: idmtools_platform_file/idmtools_platform_file/file_platform.py [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+            item: idmtools Experiment or Simulation
+            kwargs: keyword arguments used to expand functionality
+        Returns:
+            Any
+        """
+        # raise NotImplementedError("submit_job has not been implemented on the File Platform")
+
+        if isinstance(item, Experiment):
+            working_directory = self.get_directory(item)
+            result = subprocess.run(['bash', 'batch.sh'], stdout=subprocess.PIPE,
```
Comment (devclinton) [inline idmtools_platform_file/idmtools_platform_file/file_platform.py:247]: We do not want to execute the file. The file platform _produces_ files but does not execute.

**Review by devclinton — CHANGES_REQUESTED** [review]
The file platform should not actually execute anything. Otherwise, everything looks good

File: idmtools_platform_file/idmtools_platform_file/file_platform.py (hunk 1 above)
Comment (ZDu-IDM) [reply idmtools_platform_file/idmtools_platform_file/file_platform.py]: @devclinton , fixed (removed execution code totally), please verify, thanks!

**Review by shchen-idmod — APPROVED** [review]
Looks good

**Review by shchen-idmod — APPROVED** [review]
Tested again

**Review by devclinton — APPROVED** [review]

**Pull request merged**